desc/objectives/_stability.py

```diff
@@ -367,3 +369,278 @@ def compute(self, params, constants=None):
+        Lower and upper bounds on the objective. Overrides target.
+        Both bounds must be broadcastable to to Objective.dim_f. Default is ``target=0``
+    weight : {float, ndarray}, optional
+        Weighting to apply to the Objective, relative to other Objectives.
+        Must be broadcastable to to Objective.dim_f
+    normalize : bool, optional
+        Whether to compute the error in physical units or non-dimensionalize.
+        Not used since the growth rate is always normalized.
+    normalize_target : bool, optional
+        Whether target and bounds should be normalized before comparing to computed
+        values. If `normalize` is `True` and the target is in physical units,
+        this should also be set to True. Not used since the growth rate is always
+        normalized.
+    loss_function : {None, 'mean', 'min', 'max'}, optional
+        Loss function to apply to the objective values once computed. This loss function
+        is called on the raw compute value, before any shifting, scaling, or
+        normalization. Has no effect for this objective.
+    deriv_mode : {"auto", "fwd", "rev"}
+        Specify how to compute jacobian matrix, either forward mode or reverse mode AD.
+        "auto" selects forward or reverse mode based on the size of the input and output
+        of the objective. Has no effect on self.grad or self.hess which always use
+        reverse mode and forward over reverse mode respectively.
+    rho : float
+        Flux surface to optimize on. To optimize over multiple surfaces, use multiple
+        objectives each with a single rho value.
+    alpha : float, ndarray
+        Field line labels to optimize. Values should be in [0, 2pi). Default is alpha=0
+        for axisymmetric equilibria, or 8 field lines linearly spaced in [0, pi] for
+        non-axisymmetric cases.
+    nturns : int
+        Number of toroidal transits of a field line to consider. Field line
+        will run from -π*nturns to π*nturns. Default 3.
+    nzetaperturn : int
+        Number of points along the field line per toroidal transit. Total number of
+        points is ``nturns*nzetaperturn``. Default 100.
+    zeta0 : array-like
+        Points of vanishing integrated local shear to scan over.
+        Default 15 points in [-π/2,π/2]
+    lam0 : float
+        Threshold for penalizing growth rates in metric above.
+    w0, w1 : float
+        Weights for sum and max terms in metric above.
+    name : str, optional
+        Name of the objective function.
+
+    """
+
+    _coordinates = "ra"
+    _units = "(dimensionless)"
+    _print_value_fmt = "Ideal-ballooning: "
+
+    def __init__(
+        self,
+        eq,
+        target=None,
+        bounds=None,
+        weight=1,
+        normalize=True,
+        normalize_target=True,
+        loss_function=None,
+        deriv_mode="auto",
+        rho=0.5,
+        alpha=None,
+        nturns=3,
+        nzetaperturn=200,
+        zeta0=None,
+        lam0=0.0,
```

Comment:
Suggest changing this variables name to `lambda0` since it is a threshold for the growth rate, lambda

Reply:
Will do.